package com.endangeredanimals.demo.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.endangeredanimals.demo.entity.Country;

@Repository
public interface CountryRepository extends JpaRepository<Country, Integer>{
    
    Optional<Country> findByName(String countryName);
}
